assets/training/model_evaluation/components/compute_metrics/spec.yaml

```diff
@@ -69,16 +74,16 @@ outputs:
 
 is_deterministic: True
 code: ../../src
-environment: azureml://registries/azureml/environments/model-evaluation/versions/12
+environment: azureml://registries/azureml/environments/model-evaluation/versions/13
 
 command: >-
   python compute_metrics.py
-  --task ${{inputs.task}}
+  --task '${{inputs.task}}'
   --ground_truths '${{inputs.ground_truth}}'
   --predictions '${{inputs.prediction}}'
   --output '${{outputs.evaluation_result}}'
   $[[--prediction_probabilities '${{inputs.prediction_probabilities}}']]
   $[[--config-file-name '${{inputs.evaluation_config}}']]
-  $[[--ground_truths_column_name ${{inputs.ground_truth_column_name}}]]
-  $[[--predictions_column_name ${{inputs.prediction_column_name}}]]
+  $[[--ground_truths_column_name '${{inputs.ground_truth_column_name}}']]
+  $[[--predictions_column_name '${{inputs.prediction_column_name}}']]
```

Comment:
The single quotes are required to handle whitespaces

Reply:
I've added the single quotes, not removed them.